documentation/asciidoc/topics/yaml/container_extra_jvm_opts.yaml

```diff
@@ -1,3 +1,5 @@
 spec:
   container:
     extraJvmOpts: "-XX:NativeMemoryTracking=summary"
+    gossipRouterExtraJvmOpts: ""
```

Comment:
@ryanemerson Hi, could you please think about values for these two fields in this example? I'm looking for something more generic then disabling fips

Reply:
The field name is `routerExtraJvmOpts` not `gossipRouterExtraJvmOpts`.

The current `extraJvmOpts` value is valid for both the `cliExtraJvmOpts` and `routerExtraJvmOpts` fields as it's a generic JVM arg, so we can keep it simple and set it for all three.

Users may want to tune the JVM used by these components, so generic JVM args such as `-Xms4` and `-Xmx` can be specified, however the only specific use-case example we have is FIPs.

Reply:
Thank you. I went with placeholders